screens/UsersScreen/style.js

```diff
@@ -0,0 +1,59 @@
+import {StyleSheet} from 'react-native';
+import {
+  horizontalScale,
+  moderateScale,
+  verticalScale,
+} from '../../assets/metrics/Metrics';
+import {theme} from '../../assets/theme/theme';
+
+const styles = StyleSheet.create({
+  //index.js
```

Comment:
?

Reply:
I will fix it immediately after connecting to the main.